net/strongswan/files/swanctl.init

```diff
@@ -383,8 +379,14 @@ config_child() {
 
 	swanctl_xappend3 "$config_name {"
 
-	[ -n "$local_ts" ] && swanctl_xappend4 "local_ts = $local_ts"
-	[ -n "$remote_ts" ] && swanctl_xappend4 "remote_ts = $remote_ts"
+	local quoted_comma_list=""
+	config_list_foreach "$1" local_ts append_quoted_comma_list
```

Comment:
This will break things, but **yes**, we should have supported multiple values.  Although maybe people already were with:

```
     option local_ts     "192.168.1.0/24, 192.168.2.0/24, 192.168.3.0/24"
```

Are quotes strictly needed?

Reply:
I've tested the pubkeys option with quoting everything like "puba, pubb, pubc", but swanctl didn't accept it. It seems to expect "puba, "pubb", "pubc" instead. I think it is the same for local/remote_ts.

Reply:
If it really is the latter case, then we might as well use `list`.